src/components/FileDropzone.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
```

Comment:
Remove this file

Reply:
Can we do this later?  I have captured it in  my list of technical debt.

Reply:
sure